core/services/ocr2/plugins/functions/config/config.go

```diff
@@ -18,6 +23,14 @@ type PluginConfig struct {
 	ListenerEventHandlerTimeoutSec  uint32                     `json:"listenerEventHandlerTimeoutSec"`
 	MaxRequestSizeBytes             uint32                     `json:"maxRequestSizeBytes"`
 	GatewayConnectorConfig          *connector.ConnectorConfig `json:"gatewayConnectorConfig"`
+	DecryptionQueueConfig
```

Comment:
Better to go with nested fields, like for the gateway connector config. S4 will be another nested one.